.tav.yml

```diff
@@ -102,15 +89,13 @@ ioredis-v2-v4:
 ioredis-v4-v5:
   name: ioredis
   versions: '4.0.0 || 4.0.2 || 4.1.0 || 4.2.3 || 4.3.1 || 4.4.0 || 4.5.1 || 4.6.3 || 4.7.0 || 4.8.0 || 4.9.5 || 4.10.4 || 4.11.2 || 4.12.2 || 4.13.1 || 4.14.4 || 4.15.1 || 4.16.3 || 4.17.3 || 4.18.0 || 4.19.4 || 4.20.0 || 4.21.0 || 4.22.0 || 4.23.1 || 4.24.6 || 4.25.0 || 4.26.0 || 4.27.11 || ^4.28.5'
-  node: '>=6'
   commands: node test/instrumentation/modules/ioredis.test.js
   update-versions:
     mode: latest-minors
     include: '>=4.0.0 <5'
```

Comment:
nit (fine for a separate PR): Perhaps `mode: max-5` for ioredis@4. That's a lot of latest-minors that feels like more than we need to continually test.

Reply:
done!